diff --git a/node_modules/rdf-reasoner-konclude/dist/worker.js b/node_modules/rdf-reasoner-konclude/dist/worker.js
index d38823c..9c1c88d 100644
--- a/node_modules/rdf-reasoner-konclude/dist/worker.js
+++ b/node_modules/rdf-reasoner-konclude/dist/worker.js
@@ -16,7 +16,7 @@ import createKoncludeModule from "./konclude.mjs";
 // ---------------------------------------------------------------------------
 // Eager initialisation
 // ---------------------------------------------------------------------------
-const initPromise = createKoncludeModule({ print: () => { }, printErr: (msg) => console.error(msg) })
+const initPromise = createKoncludeModule({ print: () => { }, printErr: (msg) => self.postMessage({ type: "log", msg }) })
     .then((mod) => {
     self.postMessage({ type: "ready" });
     return mod;
